.github/workflows/test.yml

```
@@ -55,38 +55,58 @@ jobs:
             sleep 1
           done
           kubectl get pods -A
-      # Install kubearmor components
+      # Install kubearmor components(This will install kubearmor-client and Discovery-Engine)
```

```suggestion
      # Install KubeArmor components
```

Since it would install kubearmor itself too right?